state/state.go

```diff
@@ -683,21 +713,22 @@ func (s *State) Presence(gID discord.GuildID, uID discord.UserID) (*gateway.Pres
 		}
 
 		for _, g := range g {
-			if p, err := s.Cabinet.Presence(g.ID, uID); err == nil {
+			if p, err := s.Cabinet.Presence(g.ID, userID); err == nil {
 				return p, nil
 			}
 		}
 
 		return nil, store.ErrNotFound
 	}
 
-	return s.Cabinet.Presence(gID, uID)
+	return s.Cabinet.Presence(guildID, userID)
 }
 
 ////
 
-func (s *State) Role(guildID discord.GuildID, roleID discord.RoleID) (target *discord.Role, err error) {
-	if s.Gateway.HasIntents(gateway.IntentGuilds) {
+func (s *State) Role(guildID discord.GuildID, roleID discord.RoleID) (target *discord.Role,
+	err error) {
```

Comment:
More unnecessary indent change?

Reply:
The same here with the 100 character limit, but in this case it seems my IDE's formatter wrapped this incorrectly, so fixing this.